assets/lang/ar_AE.ini

```diff
@@ -577,6 +578,7 @@ Browse = ‎تصفح...
 Buy PPSSPP Gold = Buy PPSSPP Gold
 Choose folder = Choose folder
 Credits = ‎الحقوق
+PPSSPP Homebrew Store = PPSSPP Homebrew Store
```

Comment:
Hmm... duplicate?

-[Unknown]

Reply:
Huh, that's interesting, no idea what it did there.

Anyway I'm gonna tweak the tools a bit more and re-run, but not today.

Reply:
There's a zero-width space at the start of this line. This tool seems to be good at indirectly finding that kind of weirdness :)